org-clubhouse.el

```diff
@@ -404,6 +421,35 @@ If unset all projects will be synchronized")
                (message "%d" milestone-id)
                (funcall cb milestone-id)))))
 
+(defun org-clubhouse-prompt-for-story-type (cb)
+  (ivy-read
+   "Select a story type: "
+   (-map #'cdr org-clubhouse-story-types)
+   :history 'org-clubhouse-story-history
+   :action (lambda (selected)
+             (let ((story-type
+                    (->> org-clubhouse-story-types
+                         (-find (lambda (proj)
+                                    (string-equal (cdr proj) selected)))
+                         car)))
+               (funcall cb story-type)))))
+
+(defun org-clubhouse-prompt-for-default-story-type ()
+  (interactive)
+  (ivy-read
+   "Select a default story type: "
+   (-map #'cdr org-clubhouse-default-story-types)
+   :history 'org-clubhouse-default-story-history
+   :action (lambda (selected)
+             (let ((story-type
+                    (->> org-clubhouse-default-story-types
+                         (-find (lambda (proj)
+                                    (string-equal (cdr proj) selected)))
+                         car)))
```

Comment:
:bulb: looks like there's a common helper function waiting to be extracted here... no worries if you don| feel like it, though.